src/overlays/actors/ovl_En_Rd/z_en_rd.c

```diff
@@ -21,71 +41,858 @@ const ActorInit En_Rd_InitVars = {
+            func_80AE33F0(this);
+        }
+    } else if (this->actor.params > 0) {
+        if (this->actor.parent != NULL) {
+            func_80AE31DC(this);
+        } else {
+            this->unk_305 = 0;
+        }
+    }
+
+    if ((this->skelAnime.animCurrentFrame == 10.0f) || (this->skelAnime.animCurrentFrame == 22.0f)){
+        Audio_PlayActorSound2(&this->actor, NA_SE_EN_RIZA_WALK);
+    } else if ((globalCtx->gameplayFrames & 95) == 0) {
+        Audio_PlayActorSound2(&this->actor, NA_SE_EN_REDEAD_CRY);
+    }
+}
+
+void func_80AE2F50(EnRd* this, GlobalContext* globalCtx) {
+    SkelAnime_ChangeAnim(&this->skelAnime, &D_0600EFDC, 0.5f, 0, SkelAnime_GetFrameCount(&D_0600EFDC.genericHeader), 1, -4.0f);
+    this->unk_31B = 2;
+    EnRd_SetupAction(this, func_80AE2FD0);
+}
+
+void func_80AE2FD0(EnRd* this, GlobalContext* globalCtx) {
+    Player *player = PLAYER;
+    s32 pad;
+    s16 targetY = func_8002DAC0(&this->actor, &this->actor.initPosRot.pos);
+
+    if (func_8002DB6C(&this->actor, &this->actor.initPosRot.pos) >= 5.0f) {
+        Math_SmoothScaleMaxMinS(&this->actor.shape.rot.y, targetY, 1, 0x1C2, 0);
+    } else {
+        this->actor.speedXZ = 0.0f;
+        if (Math_SmoothScaleMaxMinS(&this->actor.shape.rot.y, this->actor.initPosRot.rot.y, 1, 0x1C2, 0) == 0) {
+            if (this->actor.params != 2) {
+                func_80AE269C(this);
+            } else {
+                func_80AE39D4(this);
+            }
+        }
+    }
+
+    Math_SmoothScaleMaxMinS(&this->unk_30E, 0, 1, 0x64, 0);
+    Math_SmoothScaleMaxMinS(&this->unk_310, 0, 1, 0x64, 0);
+    this->actor.posRot.rot.y = this->actor.shape.rot.y;
+    SkelAnime_FrameUpdateMatrix(&this->skelAnime);
+    
+    if ((!(player->stateFlags1 & 0x2C6080)) && (!(player->stateFlags2 & 0x80)) && (func_8002DB6C(&player->actor, &this->actor.initPosRot.pos) < 150.0f)) {
+        this->actor.unk_1F = 0;
+        func_80AE2B90(this, globalCtx);
+    } else {
+        if (this->actor.params > 0) {
+            if (this->actor.parent != NULL) {
+                func_80AE31DC(this);
+            } else {
+                this->unk_305 = 0;
+            }
+        }
+    }
+
+    if (this->skelAnime.animCurrentFrame == 10.0f || this->skelAnime.animCurrentFrame == 22.0f) {
+        Audio_PlayActorSound2(&this->actor, NA_SE_EN_RIZA_WALK);
+    } else if ((globalCtx->gameplayFrames & 95) == 0) {
+        Audio_PlayActorSound2(&this->actor, NA_SE_EN_REDEAD_CRY);
+    }
+}
+
+void func_80AE31DC(EnRd* this) {
+    SkelAnime_ChangeAnim(&this->skelAnime, &D_0600EFDC, 0.5f, 0, SkelAnime_GetFrameCount(&D_0600EFDC.genericHeader), 1, -4.0f);
+    this->unk_31B = 3;
+    this->unk_305 = 1;
+    EnRd_SetupAction(this, func_80AE3260);
+}
+
+void func_80AE3260(EnRd *this, GlobalContext *globalCtx) {
+    if (this->actor.parent != NULL) {
+        s32 pad;
+        s16 targetY;
+        Vec3f thisPos = this->actor.parent->posRot.pos;
+        targetY = func_8002DAC0(&this->actor, &thisPos);
```

Comment:
should there be an empty line after variables in sub-blocks too?

Reply:
I don't think it's always necessary but I do agree it's worth it here